js/src/components/pre-launch-check-item/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	CheckboxControl,
+	Panel,
+	PanelBody,
+	PanelRow,
+} from '@wordpress/components';
+import { recordEvent } from '@woocommerce/tracks';
+import { useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './index.scss';
+
+const getPanelToggleHandler = ( id ) => ( isOpened ) => {
+	recordEvent( 'pre-launch-checklist', {
+		id,
+		action: isOpened ? 'expand' : 'collapse',
+	} );
+};
+
+export default function PreLaunchCheckItem( {
+	formProps,
+	fieldName,
+	firstPersonTitle,
+	secondPersonTitle,
+	children,
+} ) {
+	const { getInputProps, setValue, values } = formProps;
+	const checked = values[ fieldName ];
+	const initialCheckedRef = useRef( checked );
+
+	if ( checked ) {
+		return (
+			<CheckboxControl
+				label={
+					initialCheckedRef.current
+						? firstPersonTitle
+						: secondPersonTitle
+				}
+				{ ...getInputProps( fieldName ) }
+				disabled
+			/>
+		);
+	}
+
+	return (
+		<div className="gla-pre-launch-checklist__checkbox">
+			<CheckboxControl { ...getInputProps( fieldName ) } />
+			<Panel>
+				<PanelBody
+					title={ secondPersonTitle }
+					initialOpen={ true }
```

Comment:
Thanks for confirming the initial state in https://github.com/woocommerce/google-listings-and-ads/pull/1612#discussion_r970181132. Adding this PR comment for following up that the `initialOpen` value should be changed to `false`.

Reply:
Just had the meeting  with the team to confirm this afternoon. Forgot to push.